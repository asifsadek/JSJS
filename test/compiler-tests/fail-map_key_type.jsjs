// key can only be num, bool, or string
val m = {
  [1,2,3]: "small list",
  [1,2,3,4,5,6,7,8,9,10]: "big list"
};
