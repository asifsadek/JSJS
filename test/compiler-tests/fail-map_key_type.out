Type Error: Cannot have 'list num' as key type in a Map.
